apps/src/sharedComponents/footer/copyright-dialog.module.scss

```diff
@@ -0,0 +1,23 @@
+@use 'sass:math';
+@import 'color.scss';
+
+.copyright {
+    h3 {
+        font-size: 28px;
+        font-weight: 600;
+        font-family: "Barlow Semi Condensed Medium", "Noto Sans", "Noto Sans Arabic", "Noto Sans Armenian", "Noto Sans Bengali", "Noto Sans SC", "Noto Sans TC", "Noto Sans Devanagari", "Noto Sans Georgian", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans Kannada", "Noto Sans Khmer", "Noto Sans KR", "Noto Sans Myanmar", "Noto Sans Sinhala", "Noto Sans Tamil", "Noto Sans Telugu", "Noto Sans Thai", "Noto Sans Thaana", sans-serif;
```

Comment:
I haven't seen this before. Is this something we should be doing in other places where we have translated text?

Reply:
fixed